tests/internal/test_wrapping.py

```diff
@@ -808,7 +807,39 @@ async def fibonacci(n):
     assert set(values) == {(n, n) for n in range(0, N)}
 
 
+@pytest.mark.subprocess()
 def test_wrapping_context_method_leaks():
+    import gc
+
+    from ddtrace.internal.wrapping.context import WrappingContext
+
+    NOTSET = object()
+
+    class DummyWrappingContext(WrappingContext):
```

Comment:
Does this new class get used anywhere?

Reply:
Yeah, let me add a comment to this.

It is used 2 lines below the diff cut off 🙄 

tl;dr; I didn't want to do `from tests.internal.test_wrapping import DummyWrappingContext` because we want to avoid over-importing too many things in this test since relies on `gc`, so can be sensitive to any background work running (our hypothesis on why it is flaky)

apologies for not including that context here.

Reply:
```suggestion
    # DEV: Redefine this module level class to avoid importing `tests.internal.test_wrapping`
    # to help reduce flakiness caused by outside impact on `gc`
    class DummyWrappingContext(WrappingContext):
```